res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+						<en>On Gnostica, Ween finds that its Council of Immortal Sages has already been kidnapped by the Shiraki. Ween must free the sages in order to be able to activate the oracle...</en>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="6" product="2" licence_type="2" guid="45e0e001-645a-449e-a861-abb17b3192b7">
+					<title>
+						<de>Lebwohl Galaxie</de>
+						<en>Farewell galaxy</en>
+					</title>
+					<description>
+						<de>Vom Orakel erfuhr Ween, die Shiraki sind Schattenwesen von der dunklen Seite der Galaxie und haben auf Codrak IV eine Vernichtungsmaschine. Sie wollen die Galaxie zerstören und nach ihren Vorstellungen wieder aufbauen. Ween muss sie stoppen...</de>
+						<en>Ween learned from the oracle that the Shiraki are shadow beings from the dark side of the galaxy and have an extermination machine on Codrak IV. They want to destroy the galaxy and rebuild it according to their ideas. Ween must stop them...</en>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="7" product="2" licence_type="2" guid="754c2dec-f377-4b4b-a24a-4b4b3a93b35b">
+					<title>
+						<de>Aliens haben mein Kindermädchen gefressen</de>
+						<en>Aliens ate my nanny</en>
+					</title>
+					<description>
+						<de>Wieder steckte Bobbys böswilliger Erzfeind aus der Schule, der einen IQ-Punkt mehr hat, hinter allem und kontrollierte all die Alien-Rassen. Und dieser macht auch einfach nicht halt und will diesmal gleich das ganze Universum zerstören. Der große Showdown beginnt...</de>
+						<en>Again, Bobby's malicious nemesis from school, who has one IQ point more, was behind everything and controlled all the alien races. And he just doesn't stop and wants to destroy the whole universe this time. The big showdown begins...</en>
+					</description>
+				</scripttemplate>
+			</children>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="437539e5-4c6b-4207-981a-051a82021745" />
+				<job index="2" function="2" required="1" gender="0" role_guid="ba05561a-0266-4cfe-b020-353781fd881c" />
+				<job index="3" function="2" required="1" gender="1" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="32" required="1" gender="1" />
+				<job index="6" function="32" required="1" gender="2" />
+			</jobs>
+
+			<genres maingenre="3" subgenres="16,1" />
+			<episodes min="8" max="8" />
+			<blocks value="1" />
+			<price min="9000" max="14000" />
+			<potential min="50" max="70" />
+			<outcome min="50" max="70" />
+			<review min="60" max="70" />
+			<speed min="50" max="70" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="c0d67fc9-1388-4715-b708-713df17c681a" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Leisure_Suit_Larry">
```

Comment:
check episode titles
* if they really need to be English (for de)
* casing in English

Reply:
They're spoofed from https://de.wikipedia.org/wiki/Leisure_Suit_Larry where some "episode" titles are also translated, some are not.

If I translate I would (like to) have to keep the alliterations etc, but ok, adding todo, maybe I (or someone else) can come up with something.